### PR TITLE
Support additional tag types in music database queries. (Fixes #29)

### LINKED ISSUE #29: Unable to list by various valid tags
Running the command:

> list title

should list all track names but it errors with:

> LocalBackend-3 ERROR    2020-04-17 23:06:06,116 Error getting distinct track: Invalid search field: track

And that's because we are incorrectly mapping MPD's 'title' tag to 'track' instead of 'track_name' but Mopidy-Local is rightly expecting 'track_name'. 

Also the tags 'track' and 'file' (and 'filename') are not supported. For example, running the command:

> list track

should list all the track numbers but it errors with:

> ACK [2@0] {list} Unknown tag type: track

Those are just missing from the mapping structure.

### PR DESCRIPTION
**Pull request by kingosticks**
Requires  https://github.com/mopidy/mopidy/pull/1900

### CODE REVIEW

**Comment by kingosticks** [comment]
I could remove the support for `comment`. I don't think there's any particular problem supporting it (except maybe multiline comments?) but it's a pretty weird/useless thing to filter or list by. And I believe the tag is disabled by default in mpd. Maybe we should just do the same.

File: tests/protocol/test_music_db.py [hunk 1]
```diff
@@ -732,15 +744,39 @@ def test_list(self):
...
+            "musicbrainz_albumid",
+            "musicbrainz_artistid",
+            "musicbrainz_trackid",
+            "performer",
+            "title",
+            "track",
+        ]
+        for field in fields:
+            self.send_request(f'list "{field}"')
+            self.assertInResponse("OK")
```
Comment (jodal) [inline tests/protocol/test_music_db.py:778]: I think this should work:

```suggestion
    @pytest.mark.parametrize('field', [
        "album",
        "albumartist",
        "artist",
        "comment",
        "composer",
        "date",
        "disc",
        "file",
        "filename",
        "genre",
        "musicbrainz_albumid",
        "musicbrainz_artistid",
        "musicbrainz_trackid",
        "performer",
        "title",
        "track",
    ])
    def test_list_all_supported_fields(self, field):
        self.send_request(f'list "{field}"')
        self.assertInResponse("OK")
```

Comment (kingosticks) [reply tests/protocol/test_music_db.py:778]: Sadly, no, [you can't use parametrize with unittest.TestCase subclasses](https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses).

**Comment by kingosticks** [comment]
Tests fail without mopidy/mopidy#1900. Will need to make this require Mopidy >= v3.3

**Pull request merged**